GUI.cpp

```diff
@@ -1637,8 +1720,12 @@ void paintboard()
         blue
     };
 
-    std::unordered_set<std::string> infoStates = { "INFO_PANEL", "INFO_PANEL/ANNOUNCEMENTS","INFO_PANEL/KEYBINDS","INFO_PANEL/SETTINGS" };
-    addSimpleButton(stonesenseState.ssState.ScreenW - (3 * TILE_WIDTH), 0, 3 * TILE_WIDTH, 3 * TILE_HEIGHT, "i", buttonColors::blue, action_toggleinfopanel, { "DEFAULT" });
+    UIStateSet infoStates;
+    infoStates.set(static_cast<size_t>(StonesenseState::UIState::INFO_PANEL));
```

Comment:
suggest using an unscoped enum for `UIState` here, to avoid the need to use explicit casts (as unscoped enums are implicitly convertible to their underlying type)